bsp/esp32_azure_iot_kit/include/bsp/esp32_azure_iot_kit.h

```diff
@@ -6,7 +6,8 @@
 
 /**
  * @file
- * @brief ESP BSP: Azure IoT kit
+ * @brief ESP BSP: ESP32-Azure IoT Kit
+ * @deprecated This board is deprecated and it is not maintanance more
```

Comment:
Nitpick:
```suggestion
 * @deprecated This board is deprecated and no longer maintained
```

In case you're interested, there are also nice 'maintenance status' badges available https://github.com/espressif/esp-usb/blob/master/host/class/uvc/usb_host_uvc/README.md?plain=1#L4